library/general/src/modules/Directory.rb

```diff
@@ -117,13 +118,24 @@ def Directory
       nil
     end
 
+    # Find an existing file in any of the YaST data directories
+    #
+    # @param relative_path [String] name of the file to find
+    # @return [String] full path of the file if it's found, nil otherwise
+    def find_data_file(relative_path)
+      possible_paths = Yast.y2paths.map { |p| File.join(p, "data", relative_path) }
+      possible_paths.find { |p| File.exist?(p) }
```

Comment:
Question is if we should use absolute `File.exist?` or something that respect SCR like `FileUtils.Exist`. I am not sure as you maybe want both...maybe option with some reasonable default?